components/Home/Hero/__tests__/index.test.tsx

```diff
@@ -0,0 +1,92 @@
+import type { NodeRelease } from '@/types';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import Hero from '..';
+import { createNodeReleases } from '@/components/__fixtures__/nodeReleases';
+
+jest.mock('../../../../hooks/useDetectOS');
+
+const releases = createNodeReleases();
+const currentLTS = releases.find(release => release.isLts) as NodeRelease;
+const currentRelease = releases.find(
+  release => release.status === 'Current'
+) as NodeRelease;
+
+describe('Hero component', () => {
+  it('renders the title', () => {
+    render(
+      <IntlProvider locale="en" onError={() => {}}>
+        <Hero
+          title="Welcome to Node.js"
```

Comment:
I feel that both the subtitle and title should not be an argument of the Hero component but rather a FormattedMessage directly within the component itself?